src/_tutorials/server/cmdline.md

```diff
@@ -73,156 +82,84 @@ The Dart VM supports many options.
-  if (paths.isEmpty) {
-    // No files provided as arguments. Read from stdin and print each line.
-    <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Using await" data-content='The "await" keyword, which works only in async functions, makes the program pause until the following expression completes.'>await</a> <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Standard I/O streams" data-content="This line reads from the standard input stream and pipes the data to the standard output stream.">stdin.pipe(stdout);</a>
-  } else {
-    for (var path in paths) {
-      int lineNumber = 1;
-      <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Create a File object" data-content="Use the File class to represent the file on the native file system.">Stream lines = new File(path)</a>
-          <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Open a file for reading" data-content="Read the content from the file asynchronously.">.openRead()</a>
-          <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Data converters" data-content="Convert data as it becomes available on the stream.">.transform(utf8.decoder)</a>
-          .transform(const LineSplitter());
-      <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Exception handling" data-content='Function calls that might generate an exception are placed in a "try" block.'>try</a> {
-        <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Get data from the stream"
-          data-content='Use "await for" to loop over the values as they become available on the stream. The program pauses while waiting for the next line.'
-        >await for (var line in lines)</a> {
-          if (showLineNumbers) {
-            stdout.write('${lineNumber++} ');
-          }
-          stdout.writeln(line);
-        }
-      } <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Exception handling" data-content='Any errors encountered in the stream are handled in the "catch" block.'>catch</a> (_) {
-        await _handleError(path);
-      }
-    }
-  }
-}
+Run the app from the command line:
 
-Future _handleError(String path) async {
-  if (<a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Test the path"
-    data-content='You can get information about the file system on which your program is running. The "await" keyword causes error handling to pause until the path is available.'
-  >await FileSystemEntity.isDirectory(path)</a>) {
-    stderr.writeln('error: $path is a directory');
-  } else {
-    <a tabindex="0" role="button" class="highlight" data-toggle="popover" title="Exit code"
-      data-content="A well-behaved command-line app sets an exit code to indicate whether the program was successful."
-    >exitCode = 2;</a>
-  }
-}
+```terminal
+$ dart dcat.dart -n dcat.dart
```

Comment:
I found this confusing (see comments for a possible solution).

Reply:
Ok, I'll revert back to using the quotes.txt file -- since that is what we had before, and no one actually complained about the missing file :).

If you'd rather that we find another solution, let me know. I think that providing the sources to our examples is orthogonal, and could be addressed in the context of curating / consolidating this and other repos (like dart-tutorials-samples).